packages/grafana-data/src/utils/nodeGraph.ts

```diff
@@ -20,8 +20,16 @@ export enum NodeGraphDataFrameFieldNames {
 
   // Id of the source node [required] [edges]
   source = 'source',
+  // The source name [required] [edges]
+  sourceName = 'sourceName',
```

Comment:
I don't think this should be here. This is definition of the requirements of node graph to render and node graph can be used for various usecases not just service graph. Also these are not really required for rendering, they are required only for the link which is already service map specific. There should not be a problem if we just don't define it here, is there? You should be able to jest send additional fields in the frame to use for links without having to declare them here.

Reply:
You're right! It felt weird to me having to modify this file, I'll remove it and keep them only in the `graphTransform` file 👍